### PR TITLE
fix bug in grouped_voucher_discounts

### PR DESCRIPTION
**Pull request by pjstevns**
this fixes an AttributeError

### CODE REVIEW

**Comment by sasha0** [comment]
Thanks for the contribution! Could you please add a test?

**Comment by pjstevns** [comment]
requested test added

File: tests/integration/offer/test_results.py [hunk 1]
```diff
@@ -28,3 +29,21 @@ def test_can_filter_post_order_actions(self):
...
+        result1 = models.BasketDiscount(D('2.00'))
+
+        offer2 = models.ConditionalOffer()
+        offer2.set_voucher(voucher)
+        result2 = models.BasketDiscount(D('1.00'))
+
+        self.applications.add(offer1, result1)
+        self.applications.add(offer2, result2)
+        discounts = self.applications.grouped_voucher_discounts
+        discounts = [*discounts,]
```
Comment (sasha0) [inline tests/integration/offer/test_results.py]: This breaks tests for Python<3.5.

Comment (pjstevns) [inline tests/integration/offer/test_results.py]: fixed

**Comment by sasha0** [comment]
Looks good, thanks.

**Pull request merged**